adding-a-new-module/template/src/commonMain/kotlin/README.md

```diff
@@ -0,0 +1 @@
+## Here  common sources
```

Comment:
Can we not just put in the main README (for adding a new module) what these paths mean?

Reply:
I would prefer to have something in this directory. would like to have a project structure created (don't only describe) in the template - it's why it calls template.
Not sure how the Gradle project will behave if it will have reference to a not existing directory

Reply:
Ah the empty directory problem.

If we have a guide in /adding-a-new-module/something.md we could have these file and link to (a section in) that file. That way the README has 2 uses, it points you to the right central documentation, and it keeps the directory.

Of course a single of text before the link (as you have in this file) is not a bad idea either, it need a little more work than `Here  common sources` though.

Reply:
Not sure why the README files are now removed after I agreed to keep them because of the empty directory problem?

Reply:
`touch .keep` ?

Reply:
As I understand you wont to remove them or update text inside. So I'm not sure what can I add in a file that exists only to be deleted, so just removed it.
Good point from @avdyushin

Reply:
`.keep` is a common solution, but please read carefully what I actually commented on this. A small README with a link to a section of the main README is perfectly fine with me.

This is what we need to do for the larger READMEs anyway, and would solve both problems (keeping the directory, and explaining what it is for) without creating long copy/pasted text everywhere that goes out of date.

Reply:
Okey. Yes, it sounds good. I'll update README